### DISCUSSION TITLE
feature: generate use-wagmi for Wagmi v2 packages

### DISCUSSION THREAD

**Discussion by apuigsech** [unicape/use-wagmi#112] [Ideas]
I have recently ported a [project](https://github.com/apuigsech/openlotto-frontend/tree/wagmi) form **wagmi v1** to **wagmi v2** using `use-wagmi`. 

Everything works fine 👌 but I need to build and install `use-wagmi` packages from the git repository because there are not *release candidate* or *beta* packages anywhere ([npmjs](https://www.npmjs.com/package/use-wagmi), [github](https://github.com/unicape/use-wagmi/releases)).

Having built packages would make `use-wagmi` (for wagmi v2) integrations far easier.

**Comment by unicape** [comment]
The use-wagmi v2 version is under development, but I have been very busy recently, so it should be updated after wagmi v2 is released to the official version.

**Comment by unicape** [comment]
[wagmi-v2](https://github.com/unicape/use-wagmi/tree/wagmi-v2) Check the specific progress here. Most of it has been completed. In addition, I am very happy to accept pull.

**Comment by unicape** [comment]
Yes, it has been published

**Comment by apuigsech** [comment]
I have it already integrated into my project.
